app/js/views/home.js

```diff
@@ -26,6 +46,8 @@ app.views.Home = (function() {
 		renderMap: function() {
 
 			var $map = this.$map;
+			var map = this.map;
+			var that = this;
```

Comment:
Let's avoid using this style of passing `this` around. How about using underscore's [_.bind(function, object)](http://underscorejs.org/#bind) method instead? Example:
```js
var renderTrees = _.bind(this.renderTrees, this);
```